src/Symfony/Bridge/Twig/Node/FormThemeNode.php

```diff
@@ -16,9 +16,9 @@
  */
 class FormThemeNode extends \Twig_Node
 {
-    public function __construct(\Twig_Node $form, \Twig_Node $resources, $lineno, $tag = null)
+    public function __construct(\Twig_Node $form, \Twig_Node $resources, $only, $lineno, $tag = null)
```

Comment:
Actually it's not marked as is, and anyone could extend the parser and the node to use custom implementation.

I don't know if it's a a doable/valid use case, but this API is public for now anyway, so BC should be preserved.